erpnext/stock/doctype/item_price/item_price.json

```diff
@@ -244,42 +438,106 @@
    "unique": 0
   }, 
   {
+   "allow_bulk_edit": 0, 
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
    "columns": 0, 
-   "fieldname": "price_list_rate", 
-   "fieldtype": "Currency", 
+   "fieldname": "uom", 
+   "fieldtype": "Link", 
    "hidden": 0, 
    "ignore_user_permissions": 0, 
    "ignore_xss_filter": 0, 
-   "in_filter": 1, 
-   "in_global_search": 1, 
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "UOM", 
+   "length": 0, 
+   "no_copy": 0, 
+   "options": "UOM", 
+   "permlevel": 0, 
+   "precision": "", 
+   "print_hide": 0, 
+   "print_hide_if_no_value": 0, 
+   "read_only": 0, 
+   "remember_last_selected_value": 0, 
+   "report_hide": 0, 
+   "reqd": 0, 
+   "search_index": 0, 
+   "set_only_once": 0, 
+   "unique": 0
+  }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "default": "1", 
+   "description": "Quantity  that must be bought or sold per UOM", 
+   "fieldname": "packing_unit", 
+   "fieldtype": "Int", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "Packing Unit", 
```

Comment:
Well that's exactly a Product Bundle, not sure it's needed here

Reply:
Yeah was thinking of this too @joezsweet , thanks! Though I needed to take a closer look at this one. Since we have to incorporate it with the minimum pricing.

Reply:
What you mean by "we have to incorporate it with the minimum pricing"?

Reply:
The usecase when an Item Price is set for a minimum of 2 and has a packing of 2. That means it can only set even numbers (in multiple of 2) during transactions.

Reply:
Well erpnext will do for you using `Product Bundle`, double check it..